gnoverse/dsocial
# Exchanging messages with Gnokey Mobile

- Prepares dSocial to send and to receive Linking Parameters from the operational system.
- Updates "@gnolang/gnonative" to "3.0.0"

How to test:
1. Open dSocial and Gnokey mobile
2. Create accounts in both of them
3. Try to Post a message using dSocial
4. After pressing the Post button, dSocial will open Gnokey Mobile and expect you select a key to proceed.
5. After selecting a key in Gnokey, it will navigate back do dSocial.
6. dSocial will receive the key address you selected and will use it to create a unsigned transaction.
7. The unsigned transaction will be sent to Gnokey to be signed.
8. Gnokey requires you to select a key to sign the transaction. Use the same key you selected before.
9. The signed transaction is sent back to dSocial.
10. dSocial will broadcast the transaction to be inserted in the blockchain.
